### DISCUSSION TITLE
关于日期显示问题

### DISCUSSION THREAD

**Discussion by 37duer** [Tokinx/Wing#121] [问答]
如何才能改成显示成具体年月日呢？而不是3天前这种显示方式

**Comment by Tokinx** [comment]
主题设置里，关闭时间格式化
![image](https://github.com/Tokinx/Wing/assets/4526339/50e5a5b7-a3b9-41ff-9ad7-dabfcfd204f1)

**Comment by vpsxb** [comment]
那如果要显示 年月日和时分 呢？ 麻烦各位大佬指惑

**Reply by Tokinx** [reply]
不开启这个就可以显示年月日了
